sdk/src/vote_program.rs

```diff
@@ -66,12 +66,12 @@ impl Lockout {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub enum VoteInstruction {
-    /// Register a new "vote account" to represent a particular validator in the Vote Contract,
-    /// and initialize the VoteState for this "vote account"
-    /// * Transaction::keys[0] - the validator id
-    /// * Transaction::keys[1] - the new "vote account" to be associated with the validator
-    /// identified by keys[0] for voting
-    RegisterAccount,
+    /// Initialize the VoteState for this "vote account"
+    /// * Transaction::keys[0] - the staker id that is also assumed to be the delegate by default
+    /// * Transaction::keys[1] - the new "vote account" to be associated with the delegate
+    InitAccount,
```

Comment:
Let's set a precedent for all programs here and call this instruction `InitializeAccount` (the counterpart to `SystemInstruction::CreateAccount`)?